website/templates/project/project.mako

```diff
@@ -188,11 +188,14 @@
                 <span data-bind="css: icon"></span>
                 </p>
 
-                % if node['description'] or 'write' in user['permissions']:
-                    <p>
+
+                <p>
+                % if not node['description']:
```

Comment:
I believe we only want to show "No description" for contributors for the project (more specifically, contributors with "write" permissions). This allows contributors to click on the text to edit it.